### ISSUE TITLE
SolutionMat vs MTX64_SolutionsMat

### ISSUE THREAD

**Issue by markuspf** [gap-packages/meataxe64#13]
When running tests like https://github.com/gap-packages/meataxe64/blob/f3ac5adf5f8965d69520e64c503b10c263ea0c22/tst/solving.tst#L23 one notices that the signs of the solutions are opposite of each other, but this means that the solution returned by `MTX64_SolutionsMat` isn't a solution:

```
gap> fld := MTX64_FiniteField(191);;
gap> mat := MTX64_RandomMat(fld, 10, 10);;
gap> v := MTX64_RandomMat(fld, 1, 10);;
gap> sol := MTX64_SolutionsMat(mat, v)[2];
gap> sol * mat = v;
false
gap> sol * mat = -v;
true
```

Is this intentional, and should it be documented like this?

**Comment by stevelinton** [comment]
Nope. That's a bug. I probably only tested it in characteristic 2.

**Comment by stevelinton** [comment]
Fixed in a0fd7d332
